modules/stanford_intranet/lib/scss/base/intranet.scss

```diff
@@ -1 +1,39 @@
 @charset "UTF-8";
+@import "./stanford_intranet.config";
+
+body {
```

Comment:
is `body` necessary here?

Reply:
I found a better option. :) Thank you for pointing it out.